planner/core/physical_plans.go

```diff
@@ -1579,11 +1579,18 @@ type PhysicalExpand struct {
 	// GroupingSets is used to define what kind of group layout should the underlying data follow.
 	// For simple case: select count(distinct a), count(distinct b) from t; the grouping expressions are [a] and [b].
 	GroupingSets expression.GroupingSets
+
+	// The level projections is generated from grouping sets，make execution more clearly.
+	LevelExprs [][]expression.Expression
+
+	// The generated column names. Eg: "grouping_id" and so on.
+	GeneratedColNames []string
```

Comment:
```suggestion
	ExtraGroupingColNames []string
```
Because `GeneratedColNames` makes people think it's related to virtual/generated column.